src/assets/scripts/client/constants/themes/default/theme.js

```diff
@@ -0,0 +1,74 @@
+import { COLOR } from './colors';
+import { DATA_BLOCK_THEME } from './moduleThemes/dataBlockThemes';
+import { TERRAIN_THEME } from './moduleThemes/terrainThemes';
+import { WIND_VANE_THEME } from './moduleThemes/windVaneThemes';
+
+export const THEME_DEFAULT = {
+    AIRSPACE_FILL: COLOR.AIRSPACE_RADIUS_FILL,
+    AIRSPACE_PERIMETER: COLOR.AIRSPACE_RADIUS_STROKE,
+    BACKGROUND: COLOR.MODERATE_GREEN,
+    COMPASS_HASH: COLOR.LIGHT_SILVER,
+    COMPASS_TEXT: COLOR.WHITE,
+    CROSSHAIR_STROKE: COLOR.CROSSHAIR_STROKE,
+    DATA_BLOCK: DATA_BLOCK_THEME.DEFAULT,
+    FIX_FILL: COLOR.WHITE_05,
+    FIX_TEXT: COLOR.WHITE_05,
+    HISTORY_DOT_INSIDE_RANGE: COLOR.WHITE,
+    HISTORY_DOT_OUTSIDE_RANGE: COLOR.LIGHT_SILVER,
+    PROJECTED_TRACK_LINES: COLOR.WHITE,
+    RADAR_TARGET: COLOR.LIGHT_SILVER,
+    RADAR_TARGET_PROJECTION_ARRIVAL: COLOR.RED_06,
+    RADAR_TARGET_PROJECTION_DEPARTURE: COLOR.BLUE_06,
+    RADAR_TARGET_PROJECTION_ESTABLISHED_ON_APPROACH: COLOR.RED,
+    RANGE_RING_COLOR: COLOR.AIRSPACE_RANGE_RING_COLOR,
+    RESTRICTED_AIRSPACE: COLOR.RESTRICTED_AIRSPACE_STROKE,
+    RING_CONFLICT: COLOR.WHITE_02,
+    RING_VIOLATION: COLOR.RED,
+    RUNWAY: COLOR.WHITE_04,
+    RUNWAY_EXTENDED_CENTERLINE: COLOR.DARK_PALE_GREEN,
+    RUNWAY_LABELS: COLOR.WHITE_08,
+    SID: COLOR.BLUE_06,
+    TERRAIN: TERRAIN_THEME.DEFAULT,
+    TOP_ROW_TEXT: COLOR.WHITE_08,
+    TRAILING_SEPARATION_INDICATOR: COLOR.RED_08,
+    VIDEO_MAP: COLOR.VIDEO_MAP_STROKE,
+    WIND_VANE: WIND_VANE_THEME.DEFAULT
+};
+
+
+// -----------------------------------------------------------------------------
+//                              NEW SCOPE THEMES
+// -----------------------------------------------------------------------------
+
+const COLOR2 = {
```

Comment:
temporary :wink: a little bit ago, i had the colors in this same file. Still fiddling with the optimal way to lay it all out. will continue on breaks at work today, leaving shortly.